### PR TITLE
hw-mgmt: thermal: Add dynamic minimum table for SN4800

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
Move SN4800 from default to real table.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-thermal-control.sh [hunk 1]
```diff
@@ -458,6 +459,29 @@ c2p_dir_untrust_t12=(15000 12 25000 13 35000 14 45000 16 $max_amb 16 )
...
+#  0-5		20	20	20	20	20	20
+#  5-10		20	30	20	20	20	30
+# 10-15		20	30	20	20	20	30
+# 15-20		20	30	20	20	20	30
+# 20-25		20	40	20	20	20	40
+# 25-30		30	50	20	20	30	50
+# 30-35		30	50	20	20	30	50
+# 35-40		40	60	20	20	40	60
+
+p2c_dir_trust_t12=(25000 12 35000 13 40000 14 $max_amb 14 )
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management-thermal-control.sh:478]: Missprint in names  t12 -> t13:
p2c_dir_trust_t12  -> p2c_dir_trust_t13
p2c_dir_untrust_t12 ...
c2p_dir_trust_t12 ...
c2p_dir_untrust_t12
unk_dir_untrust_t12

**Comment by sholeksandr** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Pull request merged**